src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -160,6 +160,10 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
         const callMemberships: CallMembership[] = [];
         for (const memberEvent of callMemberEvents) {
             const content = memberEvent.getContent();
+
+            // Dont even bother about empty events (saves us from costly type/"key in" checks in bigger rooms)
+            if (Object.keys(content).length === 0) continue;
+
             let membershipContents: any[] = [];
             // We first decide if its a MSC4143 event (per device state key)
```

Comment:
Oh, we are really close to the perf limit here.
removing this lets the room 200ms test fail.
I am not sure what data is used in this test. If its mostly empty call member events that makes sense.
But adding this initial check is okay anyways.

We see if its empty - continue
We check if there is a membership array - add it to the list of "to parse" membership data
We check if its a session format - add it to the list of "to parse" membership data
We check if there is any "to parse" membership data

this seems like a reasonable approach.

Reply:
Oh well this code is not even run. the test just seems to be super flaky.